pkg/crc/preflight/preflight_checks_tray_windows.go

```diff
@@ -100,8 +100,12 @@ func fixTrayInstalled() error {
 }
 
 func removeTray() error {
+	trayShortcutPath := filepath.Join(constants.StartupFolder, constants.TrayShortcutName)
 	_ = stopTray()
-	return goos.Remove(filepath.Join(constants.StartupFolder, constants.TrayShortcutName))
+	if os.FileExists(trayShortcutPath) {
```

Comment:
Please refactor to a single function that gets re-used:

```
RemoveFileIfExists(filepath string) err {
    if (filepath) {
       return goos.Remove(filepath)
    }
    return nil
}
```

Reply:
Changed